js/common/charts.d.ts

```diff
@@ -277,3 +281,57 @@ export type WordWrap = 'normal' | 'breakWord' | 'none';
  * @namespace DevExpress.common.charts
  */
 export type ZoomPanAction = 'zoom' | 'pan';
+
+/**
+ * @docid
+ * @type object
+ * @public
+ * @namespace DevExpress.common.charts
+ */
+export type ChartsColor = {
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    base?: string;
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    fillId?: string;
+};
+
+/**
+ * @docid viz.registerGradient
+ * @publicName registerGradient(type, options)
+ * @static
+ * @public
+ */
+export function registerGradient(type: string, options: { rotationAngle?: number; colors: Array<GradientColor> }): string;
```

Comment:
Since `function` is a value, runtime that is in that `graphic_objects.js` must also be moved here